File: sdkwork-backend-sdk-go/types/mime_type.go
package types


type MimeType struct {
	Type string `json:"type"`
	Subtype string `json:"subtype"`
	Parameters map[string]string `json:"parameters"`
	Charset string `json:"charset"`
	Concrete bool `json:"concrete"`
	SubtypeSuffix string `json:"subtypeSuffix"`
	WildcardType bool `json:"wildcardType"`
	WildcardSubtype bool `json:"wildcardSubtype"`
}

File: sdkwork-backend-sdk-csharp/Models/MimeType.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class MimeType
    {
        public string? Type { get; set; }
        public string? Subtype { get; set; }
        public Dictionary<string, string>? Parameters { get; set; }
        public string? Charset { get; set; }
        public bool? Concrete { get; set; }
        public string? SubtypeSuffix { get; set; }
        public bool? WildcardType { get; set; }
        public bool? WildcardSubtype { get; set; }
    }
}
